app/src/main/java/com/android/voyageur/ui/trip/Tabs.kt

```diff
@@ -81,16 +83,17 @@ fun TopTabs(
         Tab(
             selected = navigationActions.getNavigationState().currentTabIndexForTrip == index,
             onClick = { navigationActions.getNavigationState().currentTabIndexForTrip = index },
-            text = { Text(title) })
+            text = { Text(title, fontSize = 12.sp) })
```

Comment:
Here, as discussed in the meeting, maybe we'll put icons instead of the descriptions of the tabs, as on small screens, the writing might be too small or overlap, but I think we'll include this in a UI improvements PR.

Reply:
Ok